settings/src/components/account-status.js

```diff
@@ -13,24 +13,22 @@ import ScreenLink from './screen-link';
 
 /**
  * Render the Account Status.
+ *
+ * @param root0
+ * @param root0.totpEnabled
+ * @param root0.backupCodesEnabled
+ * @param root0.webAuthnEnabled
```

Comment:
I know this is what we get with autofix, but I think `props` is more meaningful
```suggestion
 * @param props
 * @param props.totpEnabled
 * @param props.backupCodesEnabled
 * @param props.webAuthnEnabled
```

Reply:
Commit added: [752ec39](https://github.com/WordPress/wporg-two-factor/pull/197/commits/752ec39d821c186680af7fe4a28be9f6d6b5decd)